apps/src/fish/FishView.jsx

```diff
@@ -154,7 +154,7 @@ class FishView extends React.Component {
 
     return (
       <StudioAppWrapper>
-        <CodeWorkspaceContainer topMargin={0}>
```

Comment:
How is FishView related to the rest of this change?

Reply:
`CodeWorkspaceContainer` used to require the `topMargin` prop, but no longer does (it will set `top` to 0 by default, which is how it was used here), so i removed the prop here